ExtIO_sddc/r2iq.cpp

```diff
@@ -72,11 +72,10 @@ int r2iqControlClass::Setdecimate(int dec)
 
 float r2iqControlClass::setFreqOffset(float offset)
 {
-	this->mtunebin = int(offset * halfFft / mratio[mdecimation] + 0.5);
-	float delta = offset - ((float)this->mtunebin * mratio[mdecimation] / halfFft);
-	float ret = -delta / mratio[mdecimation];
-	DbgPrintf("tunebin : %f -> %d delta %f (%f)\n", offset, this->mtunebin, delta, ret);
-
+	this->mtunebin = int(offset * halfFft/4  + 0.5 ) * 4;  // mtunebin step 4 bin  ? 
+	float delta = offset - ((float)this->mtunebin  / halfFft);
+	float ret = - delta * mratio[mdecimation]; // ret increases with higher decimation
+	DbgPrintf("LOfreq %d mtunebin %d delta %f (%f)\n", (int) (offset * ADC_FREQ/2), this->mtunebin, delta, ret);
```

Comment:
can you print the LOfrq in RadioHandler? I don't want r2iq depends on ADC_FREQ. I am working on a change to have a dynamic ADC frequency.

Reply:
Yes. I'll do. Dynamic ADC frequency is a good feature !